### PR TITLE
Input

### PR DESCRIPTION
**Pull request by arpit01923**
- Input component is created

1. we have three input one is text, second is password and last is submit input
2. we put require field on text and password component
3. submit should only work if both text and password field is filled
image
image

### CODE REVIEW

**Review by laxmikanthkonthum — APPROVED** [review]
Looks good to me

File: css/example/Input.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            <input class="input" type="text" name="username" required>
+        </div>
+
+        <div class="input-password">
+            <label for="password">Password</label>
+            <input class="input" type="password" name="password" required>
+        </div>
+
+        <div class="input-submit">
+            <a href="#"><input class="input" type="submit" name="submit"></a>
```
Comment (bajajcodes) [inline css/example/Input.html]: I think there is no need of input tag inside hyperlink tag(a).

File: css/components/Input.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.input-name,.input-password,.input-submit{
+    margin: 1rem;
+}
+
+.input-submit{
+    text-align: center;
+}
+
+.input-name label,.input-password label,.input-submit label{
```
Comment (bajajcodes) [inline css/components/Input.css]: Use class selectors here to apply css.

**Review by HarshithaKonduru214 — APPROVED** [review]

File: css/example/Input.html (hunk 1 above)
Comment (arpit01923) [reply css/example/Input.html]: removed (a) tag

File: css/components/Input.css (hunk 2 above)
Comment (arpit01923) [reply css/components/Input.css]: class name label should be used

**Pull request merged**